Update backing up and restoring page of the accounts section for the … (#3121)
w3f/polkadot-wiki: docs/learn/learn-account-restore.md

same seed words will have control over that account on-chain. This is why it is so important to keep
your seed words secret and safe.

Step 1

> ![restore-using-json](../assets/accounts/backing_up_and_restoring/via_existing_seed_1.png)

Reviewer: The alt text for the three newly added images should provide context for the image. Right now it is restore-using-json for all the three image files

![restore-using-parity](../assets/parity_Signer_restore2.PNG)

The identity has now been recovered and you can select a network to create the first account.

## Transferring Polkadot-JS Apps Accounts/Addresses From One Computer to Another

> **Note:** Note: This will overwrite any existing accounts with the same pubkey on your new computer. 
> This generally should not make a difference (since it can still access the same account), but might if you 
> have e.g. an account which was stored externally in the extension on the old computer but was created 
> directly in browser on the new one.

This has been tested on Brave and Chrome, but not other browsers.

1. Go to Polkadot-JS Apps

> 2. Go to JavaScript console (on Brave, NOT on Polkadot-JS)

Reviewer: ```suggestion
2. Go to JavaScript console on the browser (Available in Developer Tools)
```

> 3. Type in the command JSON.stringify(localStorage)

Reviewer: Just the way code is shown distinct from the text in the steps below, ensure this line below is displayed as code

JSON.stringify(localStorage)

> 4. Copy and paste the returned string to a text editor.

Reviewer: ```suggestion
4. Copy and paste the returned string to a text editor and save the file.
```

5. Check that the string you pasted both begins and ends with a tick mark ('). If not, add one to the beginning and end.

> 6. Save and send that JSON file to the new computer.

Reviewer: ```suggestion
6. Save and send that file with the copied string to the new computer.
```

7. On new computer, go to Polkadot-JS Apps

> 8. Open the Javascript console

Reviewer: ```suggestion
8. Open the Javascript console on the browser (Available in Developer Tools)
```

> 9. Set a variable raw equal to the JSON you created =

Reviewer: ```suggestion
9. Set a variable raw equal to the string from the text file
```

```
raw = ... copy-pasted json from original computer ...
```

> 10. Run the following code:

Reviewer: ```suggestion
10. Run the following code on the console:
```
